frontend/test/components/search/shared/searchForm.spec.ts

```diff
@@ -289,6 +295,49 @@ describe("Documentunit Search", () => {
     expect(
       screen.getByLabelText("Dokumentationseinheiten mit Dublettenverdacht"),
     ).not.toBeChecked()
+    expect(screen.getByLabelText("Nur mir zugewiesen")).not.toBeChecked()
+  })
+
+  test(`resets all docoffice specific filter, when 'Nur meine Dokstelle Filter' is unchecked`, async () => {
+    const { user } = renderComponent(Kind.DECISION)
+
+    await user.click(screen.getByLabelText("Nur meine Dokstelle Filter"))
+    await user
+      .click(screen.getByLabelText("Prozessschritt"))
+      .then(async () => await user.click(screen.getByLabelText("Step A")))
+    await user.type(
+      screen.getByLabelText("jDV Übergabedatum Suche"),
+      "11.11.2011",
+    )
+    await user.click(screen.getByLabelText("Terminiert Filter"))
+    await user.click(
+      screen.getByLabelText("Nur fehlerhafte Dokumentationseinheiten"),
+    )
+    await user.click(
+      screen.getByLabelText("Dokumentationseinheiten mit Dublettenverdacht"),
+    )
+    await user.click(screen.getByLabelText("Nur mir zugewiesen"))
+    expect(screen.getByLabelText("Nur mir zugewiesen")).toBeChecked()
+
+    await user.click(screen.getByLabelText("Nur meine Dokstelle Filter"))
+    expect(
+      screen.getByLabelText("Nur meine Dokstelle Filter"),
+    ).not.toBeChecked()
+    await user.click(screen.getByLabelText("Nur meine Dokstelle Filter"))
+    expect(screen.getByLabelText("Nur meine Dokstelle Filter")).toBeChecked()
+
+    expect(screen.getByLabelText("jDV Übergabedatum Suche")).toHaveValue("")
+    expect(screen.getByLabelText("Prozessschritt").textContent).equals(
+      "Nicht ausgewählt",
+    )
+    expect(screen.getByLabelText("Terminiert Filter")).not.toBeChecked()
+    expect(
+      screen.getByLabelText("Nur fehlerhafte Dokumentationseinheiten"),
+    ).not.toBeChecked()
+    expect(
+      screen.getByLabelText("Dokumentationseinheiten mit Dublettenverdacht"),
+    ).not.toBeChecked()
+    expect(screen.getByLabelText("Nur mir zugewiesen")).not.toBeChecked()
```

Comment:
Ist das nicht der gleiche Test case wie der darunter (`resets own doc office fields when check box is unchecked`)?

Reply:
gut gesehen, den hatte ich übersehen und dachte der fehlt noch. hab ihn wieder weggenommen